Description: Allow use of first sector on btrfs
Author: Colin Watson <cjwatson@ubuntu.com>
Bug-Ubuntu: https://bugs.launchpad.net/bugs/757446
Forwarded: yes
Applied-Upstream: http://bazaar.launchpad.net/~vcs-imports/grub/grub2-bzr/revision/3207
Last-Update: 2011-04-11

Index: b/grub-core/fs/btrfs.c
===================================================================
--- a/grub-core/fs/btrfs.c
+++ b/grub-core/fs/btrfs.c
@@ -1462,6 +1462,9 @@
     .close = grub_btrfs_close,
     .uuid = grub_btrfs_uuid,
     .label = grub_btrfs_label,
+#ifdef GRUB_UTIL
+    .reserved_first_sector = 1,
+#endif
   };
 
 GRUB_MOD_INIT(btrfs)
